interface/src/app/gene/searchMany/search.js

```diff
@@ -1,8 +1,108 @@
-angular.module('adage.gene.search.directives', [
-  'adage.gene.search.services',
-  'adage.gene.utils'
+angular.module('adage.gene.search', [
```

Comment:
The filename `search.js` and module name `search` for this seems a bit weird since, if I understand correctly, this is only part of the type of search that you plan to support. Not sure how file naming is usually done but maybe `search_many.js` or something might be more appropriate?